pytorch_lightning/lite/lite.py

```diff
@@ -203,7 +203,7 @@ def setup_dataloaders(
             for dataloader in dataloaders
         ]
         dataloaders = dataloaders[0] if len(dataloaders) == 1 else dataloaders
-        return dataloaders  # type: ignore[return-value]
```

Comment:
mypy was complaining about these unused ignores. I'm happy to revert these deletes if they're there for a reason!